### ISSUE TITLE
Stream pointer might be NULL

### ISSUE THREAD

**Issue by QiAnXinCodeSafe** [intel/libva-utils#181]
'file_size ' is defined here, but may be NULL

https://github.com/intel/libva-utils/blob/7be46f017caee2a23097bc8371f2fe4eb1b677a0/vendor/intel/sfcsample/VDecAccelVA.cpp#L598

Stream pointer might be NULL
https://github.com/intel/libva-utils/blob/7be46f017caee2a23097bc8371f2fe4eb1b677a0/vendor/intel/sfcsample/VDecAccelVA.cpp#L599

**Comment by XinfengZhang** [comment]
agree, should add check of the sfc_stream --- FILE* sfc_stream = fopen("sfc_sample_176_144_argb.yuv", "wb");
